src/renderer/components/side-nav/side-nav.vue

```diff
@@ -25,14 +30,19 @@
         class="navOption mobileHidden"
         role="button"
         tabindex="0"
+        :title="$t('Trending')"
```

Comment:
This should be changed to `$t('Trending.Trending')` before merging (#1638 changed this)